dev/strongpoc/serializers.py

```diff
@@ -22,7 +22,7 @@ class Meta:
 
 class PointOfContactSerializer(serializers.ModelSerializer):
     # team = serializers.StringRelatedField()
-    team = TeamSerializer()
+    # team = TeamSerializer()
```

Comment:
It was to illustrate exactly that. How if you nest a serializer, that the field is then constrainted by the validation/fields of the other serializer.